Dockerfile

```diff
@@ -10,19 +10,25 @@ ENV PYTHONPATH /root
 ARG VERSION
 ARG DOCKER_IMAGE
 
-RUN apt-get update && apt-get install build-essential -y
+# Combining RUN commands with the && : pattern at the end
+# 1. Update the necessary packages for flytekit
+# 2. Delete apt cache. Reference: https://gist.github.com/marvell/7c812736565928e602c4
+# 3. Change the permission of /tmp, so that others can run command on it
+RUN apt-get update && apt-get install build-essential libmagic1 -y \
```

Comment:
let's not install `libmagic1` on the docker image.

Reply:
no problem!

Reply:
The feature added in https://github.com/flyteorg/flytekit/pull/1893/ should be opt-in. We should make the default flytekit image as small as possible.

Reply:
Thanks a lot!